docs/UserGuide.md

```diff
@@ -355,16 +333,38 @@ _After executing command: `view 1`_
 
 ### Copy details of internship : `copy`
 
-Copies all details of internship at index into clipboard
+Your friend is asking for one of your internship application details? Copy all the details of internship application into your computer's [clipboard](#glossary).
 
```

Comment:
I think can consider:
"Quickly share your application details by copying it into your computer's clipboard." 
I'm not sure if having too many questions to start the command description is ok. I think we should ask her later.

Reply:
updated, i think should be okay, we can discuss again on thursday